source/UIAUtils.py

```diff
@@ -170,7 +170,11 @@ def __init__(self,textRange):
 		self.textRange=textRange
 
 	def getValue(self,ID,ignoreMixedValues=False):
-		val=self.textRange.getAttributeValue(ID)
+		try:
+			val=self.textRange.getAttributeValue(ID)
+		except COMError:
+			# #7124: some text attributes are not supported in  older Operating Systems 

```

Comment:
Is this something that would be helpful to log? Would that cause a lot of noise in the log? I'm considering a case where we are investigating why something does not work in a particular OS for instance.

What happens when `reservedNotSupportedValue` is returned, is this handled gracefully in all cases?